src/metabase/public_settings.clj

```diff
@@ -70,12 +70,54 @@
   :audit   :getter
   :default true)
 
+(declare site-uuid)
```

Comment:
Maybe just move the `site-uuid` setting (+ related settings) up to the top? I don't think there's really a specific order right now

Reply:
done. good point. I think i was trying to minimize the diff at first, which is not necessary at all.